mpp/include/mpp_domains_define.inc

```diff
@@ -799,7 +799,9 @@
        domain%tileList(1)%yend   = global_indices(4)
        allocate(domain%x(1), domain%y(1) )
        allocate(domain%tile_id(1))
+       allocate(domain%tile_id_all(1))
```

Comment:
If this is always just size one, why not make it an allocatable scalar?

Reply:
Not sure what you mean by allocatable scalar.  The use of "domain" in this function is for 2D domains where there is a single rectangular grid and the arrays need only one element.  The same data structure is used within the mosaic define and the original intent was to potentially have more than one mosaic tile per MPI-rank.  RIght now we don't run it that way, but the logic is there in case it is needed.

Reply:
OK, that answers my question then.